core/config/docs/chains-evm.toml

```diff
@@ -311,13 +311,20 @@ PollInterval = '10s' # Default
 # SelectionMode controls node selection strategy:
 # - HighestHead: use the node with the highest head number
 # - RoundRobin: rotate through nodes, per-request
+# - PriorityLevel: use the node with the smallest order number
 # - TotalDifficulty: use the node with the greatest total difficulty
 SelectionMode = 'HighestHead' # Default
 # SyncThreshold controls how far a node may lag behind the best node before being marked out-of-sync.
 # Depending on `SelectionMode`, this represents a difference in the number of blocks (`HighestHead`, `RoundRobin`, `PriorityLevel`), or total difficulty (`TotalDifficulty`).
 #
 # Set to 0 to disable this check.
 SyncThreshold = 5 # Default
+# SwitchToBestNodeInterval interval in which to check and force internal services to use the "best" node as
+# defined by the SelectionMode
+# Setting this to a low value might cause RPC to switch too aggressively
+#
+# Set to '0s' to disable
+SwitchToBestNodeInterval = '0s' # Default
```

Comment:
nit: i find this name too verbose, and i'm confused by the the implementation because there doesn't seem to be any switch, just dropping.

wdyt about NodeCheckInterval or BestNodeInterval?

Reply:
Feels kind of like a lease to me, and we have precendent for that in the config already.

Reply:
Leas sounds good to me, something like `NodeLeaseDuration`.

Reply:
Are we already under a `Node` context though? `NodePool.BestLeaseDuration`?

Reply:
i like `NodePool.LeaseDuration` + a good comment that the notion of "best" is leased

Reply:
Changed to `NodePool.LeaseDuration` in [`364663e` (#10726)](https://github.com/smartcontractkit/chainlink/pull/10726/commits/364663ef700cdae19beeb7cdbc8ffc91fd526e1f)